lenovo/thinkpad/x230/default.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib; {
+  imports = [
+    ../.
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop/hdd
+  ];
+
+  boot = {
+    kernelModules = [
+      "kvm-intel"
```

Comment:
Nitpick: `kvm-intel`  is recognized by `nixos-hardware-config`, so it's better to not duplicate it here.

Reply:
but this would assume that everybody starts with nixos-hardware-config, right? this config assumed "clean slate" but i can remove this part and just put it into my customized config

Reply:
Indeed, assumption is that users run `nixos-generate-config` because the long-term plan is to move to "dynamic dispatch" of sorts and to generate all of this in `nixos-generate-config` based on discovered hardware. Explicitly specifying model has its limitations: some hardware might be defunct, something may have been customized (e.g. SSD, or fingerprint sensor), or maybe our assumptions are incorrect. 

It's just that `nixos-generate-config` is not advanced enough to do that yet.